hibernate-core/src/main/java/org/hibernate/resource/beans/container/internal/JpaCompliantLifecycleStrategy.java

```diff
@@ -125,7 +125,7 @@ public void initialize() {
 				throw e;
 			}
 			catch (Exception e) {
-				log.debugf( "Error resolving CDI bean [%s] - using fallback" );
+				log.debugf( "Error resolving CDI bean [%s] - using fallback", beanType );
```

Comment:
It is really annoying for such String format parameter mismatch issue. We encountered every now and then. Sometimes it gets exposed easily, sometimes the issue is hidden simply because the statement is seldom run.